deploy/environments/applications_hpc.yaml

```diff
@@ -14,8 +14,9 @@ spack:
         - neurodamus-mousify
         - parquet-converters
         - py-basalt
+        - py-libsonata
```

Comment:
Why do we need a module for that?

Reply:
Oh, I abused this PR and added this ever-wanted module.

Reply:
Then it should go into libraries, and also listed as a spec.